src/main/java/org/radarbase/appserver/controller/ProtocolEndpoint.java

```diff
@@ -43,6 +47,10 @@ public ProtocolEndpoint(ProtocolGenerator protocolGenerator) {
   }
 
   @GetMapping("/" + PathsUtil.PROTOCOL_PATH)
+  @Authorized(
+      permission = AuthPermissions.READ,
+      entity = AuthEntities.SUBJECT,
+      permissionOn = PermissionOn.PROJECT)
```

Comment:
This not correct. If authorising on project or subject, you need both as path params in the request